src/main/java/pl/byczazagroda/trackexpensesappbackend/service/UserServiceImpl.java

```diff
@@ -0,0 +1,64 @@
+package pl.byczazagroda.trackexpensesappbackend.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+import pl.byczazagroda.trackexpensesappbackend.dto.AuthRegisterDTO;
+import pl.byczazagroda.trackexpensesappbackend.model.User;
+import pl.byczazagroda.trackexpensesappbackend.model.UserStatus;
+import pl.byczazagroda.trackexpensesappbackend.repository.UserRepository;
+
+import javax.validation.Valid;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@RequiredArgsConstructor
+@Service
+@Validated
+public class UserServiceImpl implements UserService{
```

Comment:
Code formating: Enter between class proprties and methods format the whole class